docs/index.rst

```diff
@@ -2,7 +2,7 @@ Parcels documentation
 =====================
 
 
-Welcome to the documentation of Parcels. **Parcels** (Probably A Really Computationally Efficient Lagrangian Simulator) is a set of Python classes and methods to create customisable particle tracking simulations using output from Ocean Circulation models. Parcels can be used to track passive and active particulates such as water, plankton, `plastic <http://www.topios.org/>`_ and `fish <https://github.com/Jacketless/IKAMOANA>`_.
+Welcome to the documentation of Parcels. **Parcels** provides a set of Python classes and methods to create customisable particle tracking simulations using gridded output from (ocean) circulation models. Parcels can be used to track passive and active particulates such as water, plankton, `plastic <http://www.topios.org/>`_ and `fish <https://github.com/Jacketless/IKAMOANA>`_.
```

Comment:
> set of Python classes and methods to create

A bit off-tangent for this PR, but thoughts on "modelling framework"?

Reply:
> because they typically think of a modelling framework as a black-box code

I see. From a software engineering/data science POV hearing "framework" I think of something like Pandas (dataframes) and Sklearn (fit-predict) APIs that offer a lot of flexibility but provide a structured way of thinking about the problem.

> So I'd prefer to keep as is
Sounds good. Our main target demographic is oceanographers anyway :)